aptos-move/move-examples/post_mint_reveal_nft/Move.toml

```diff
@@ -9,4 +9,6 @@ AptosToken = { local = "../../framework/aptos-token" }
 DataStructure = { local = "../data_structures" }
 
 [addresses]
-source_addr = "_"
+aptos_framework = "0x1"
+source_addr = "0xcafe"
+post_mint_reveal_nft = "0xc3bb8488ab1a5815a9d543d7e41b0e0df46a7396f89b22821f07a4362f75ddc5"
```

Comment:
Is this for unit test?

Reply:
yeah this is for developers running `aptos move test` without having to add addresses

Reply:
Does this affect publishing at all? I remember there might be some issue with replacing the addresses if they're not set to _

Reply:
yeah good point, I think if they publish then they need to swap this with their real addresses. I just add some comments in the `Move.toml` file to clarify this.